tests/test_plotting.py

```diff
@@ -0,0 +1,180 @@
+"""Tests for the plotting module."""
+from pathlib import Path
+
+import pandas as pd
+import pytest
+
+from topostats.plotting import (
+    importfromfile,
+    savestats,
+    # pathman,
+    # labelunitconversion,
+    # dataunitconversion,
+    plotkde,
+    plotkde2var,
+    plothist,
+    plothist2var,
+    plotdist,
+    plotdist2var,
+    plotviolin,
+    # plotjoint,
+    # plotLinearVsCircular,
+    computeStats,
+)
+
+BASE_DIR = Path.cwd()
+RESOURCES = BASE_DIR / "tests" / "resources"
+
+
+def test_importfromfile(minicircle_all_statistics: pd.DataFrame) -> None:
+    """Regression test for importfromfile()."""
+    minicircle_df = importfromfile(RESOURCES / "minicircle_default_all_statistics.csv")
+    assert isinstance(minicircle_df, pd.DataFrame)
+    pd.testing.assert_frame_equal(minicircle_all_statistics, minicircle_df)
+
+
+def test_savestats(minicircle_all_statistics: pd.DataFrame, tmpdir) -> None:
+    """Regression test for savestats()."""
+    savestats(str(tmpdir), minicircle_all_statistics)
+    assert Path(str(tmpdir)[:-5] + "_evaluated.txt").is_file()
+
+
+# Not testing, path is contingent on system and changes with each test
+# def test_pathman(tmpdir) -> None:
+#     """Regression test for pathman()."""
+#     plotname = pathman(str(tmpdir))
+#     assert plotname == "/tmp/"
+
+
+def test_labelunitconversion() -> None:
```

Comment:
Is this something to be completed later, or is it functional currently?

Reply:
Well spotted, just a place holder for now.